src/transform.test.ts.md

```diff
@@ -300,7 +304,49 @@ Generated by [AVA](https://avajs.dev).
             a = _a;␊
         }␊
     }␊
-    `
+    ␊
+    contract Foo8 {␊
+    ␊
+        modifier hasModifier(){␊
+            _;␊
+        }␊
+    ␊
+        function __Foo8_init() internal onlyInitializing {␊
+            __Foo8_init_unchained();␊
+        }␊
+    ␊
+        function __Foo8_init_unchained() internal onlyInitializing {␊
```

Comment:
I've realized there is a preexisting bug: the modifier was removed!

Reply:
Note, the modifier might have side effects, so it should NOT be added to both the chained and the unchained versions.
I suggest it goes in the unchained.

Reply:
Yes that is what we will do, include the custom modififers only on the unchained method, that will not be included on this PR but in a following one